docs/adminguide/updating/index.rst

```diff
@@ -0,0 +1,49 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+.. _doc_adminguide_updating:
+
+.. cspell:words linenos literalinclude
+
+======================
+Parsec Server updating
+======================
+
+This guide covers the updating procedure for the `Parsec`_ server (also called `parsec-backend`).
+
+.. note::
+
+  We consider that you are using the same infrastructure configuration as in the :ref:`Parsec Server installation guide <doc_adminguide_hosting>`.
+  Adapt this guide to your own situation.
+
+.. _Parsec: https://parsec.cloud
+
+Updating to a newer version
+***************************
+
+.. warning::
+
+  Before updating to a newer version, we recommend that you backup the database,
+  In case you need to rollback the update.
```

Comment:
BTW we should probably add a section below instructing HOW to rollback in case of problems.

Reply:
> we should probably add a section below instructing HOW to rollback in case of problems.

I've planned for that, but the server don't provide a way to rollback (undo the applied database migration that was applied during the update).
That would require the user to preemptively create a database backup (the goal of this text) and use that backup when needed.

But the user will lose data (Lose the data that was added after the update).

Reply:
done